kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -63,6 +65,7 @@ export default function useQuizCreation() {
    * @throws {TypeError} if section is not a valid QuizSection
    **/
   function updateSection({ section_id, ...updates }) {
+    set(quizHasChanged, true);
```

Comment:
I think probably the more likely case is for someone to change something, then change it back - in which case this would be set to true, and there not actually be any changes. I think that edge case is worth having for the sake of simplicity (rather than deep diff checking).